apps/contributor/test/contributor.js

```diff
@@ -0,0 +1,170 @@
+const namehash = require('eth-ens-namehash').hash;
```

Comment:
we should be able to use the namehash from ethers:
`const namehash = require('ethers').utils.namehash;`
as ethers.js is already a dependency we do not need to add another dependency

Reply:
cool will change that

Reply:
for the ganache script, it is basically the same aragon one, but I change 
```
run_tests() {
  echo "Running tests..."
  npx truffle test --network rpc "$@"
}
```
to 
```
run_tests() {
  echo "Running tests..."
  npx aragon contracts test --network rpc "$@"
}
```
Because `npx truffle test --network rpc "$@"` didn't work for me for no reason...